### ISSUE TITLE
Subscription config for Clash includes names of VLESS proxy configs in the proxy group, but no VLESS configs in the proxies list

### ISSUE THREAD

**Issue by fapspirit** [Gozargah/Marzban#1356]
**Describe the bug**
Subscription config which is available via subscription link for Clash does not include configs with VLESS protocol in the `proxies`, but names of VLESS configs are presented in the `proxy-groups[].proxies` array. This makes Clash for Windows to return a error when adding servers via subscription link since there are no VLESS configs in the `proxy` array.

**To Reproduce**
1. Add VLESS and VMess proxy configs in the dashboard
2. Create user with both VLESS and VMess configs enabled
3. Copy subscription link (.e.g. `https://example.com/sub/token123`)
4. Make request to get subscription config for clash: `curl https://example.com/sub/token123/clash`, you'll see somthing like this:
```yaml
mode: Global
port: 7890
proxies:
- name: Marz (mee) [Trojan - ws]
  network: ws
  password: 123123123
  port: 2083
  server: example.com
  sni: google.com
  tls: true
  type: trojan
  udp: true
  ws-opts: {}
- cipher: chacha20-ietf-poly1305
  name: Marz (mee) [Shadowsocks - tcp]
  network: tcp
  password: 123123123
  port: 1080
  server: example.com
  tcp-opts: {}
  type: ss
  udp: true
rules: []
proxy-groups:
- name: ♻️ Automatic
  type: url-test
  url: http://www.gstatic.com/generate_204
  interval: 300
  proxies:
  - Marz (mee) [VLESS - tcp]
  - Marz (mee) [VLESS - grpc]
  - Marz (mee) [Trojan - ws]
  - Marz (mee) [Shadowsocks - tcp]
```

**Expected behavior**
VLESS configs are not presented in `proxies` and their names are not presented in the `proxy-groups[].proxies` array as well since VLESS is not yet supported by Clash for Windows.

**Screenshots**
Screenshot of the error
![telegram-cloud-photo-size-2-5204165895394552188-x](https://github.com/user-attachments/assets/56227f89-9c81-4a09-90a8-cc8114e23bc2)

**Machine details (please complete the following information):**
 - OS: [e.g. ubuntu 20]: N/A
- Python version: [e.g 3.8]: N/A
- Nodejs version: [e.g 16.17]: N/A
- Browser [e.g. chrome, safari]: N/A

**Additional context**

**Comment by fapspirit** [comment]
Looks like problem lies here:
https://github.com/Gozargah/Marzban/blob/49accabb31a094cf0c18223b3fbeade923f9bd6d/app/subscription/clash.py#L180
This line produces implicit side-effect by appending a proxy remark which is a value for `proxy-groups[].proxies`.
This function is invoked here:
https://github.com/Gozargah/Marzban/blob/49accabb31a094cf0c18223b3fbeade923f9bd6d/app/subscription/clash.py#L258
This function is executed for each config, but after the execution there are no branch for the `vless` protocol to add it to list of `proxies`.

I think the fix is to move the appending of remark from the `make_node` method to this line:
https://github.com/Gozargah/Marzban/blob/49accabb31a094cf0c18223b3fbeade923f9bd6d/app/subscription/clash.py#L291

**Comment by fapspirit** [comment]
Fixed in #1357
